### DISCUSSION TITLE
Is there a way to search and delete a line?

### DISCUSSION THREAD

**Discussion by f11xter** [yassinebridi/serpl#25] [Q&A]
I can't seem to match newlines (I've tried `\n` but it doesn't match anything), so whenever I replace a line with nothing, it leaves a blank line in the output.

Input fille:
```
...
line to delete
...
```

Replace `line to delete` with nothing:
```
...

...
```

Desired:
```
...
...
```

**Comment by yassinebridi** [comment]
Currently there is no way to do this,
i created an issue from this, and will fix it asap

**Comment by yassinebridi** [comment]
This will be fixed by #32
